server/src/infra/entities/user.entity.ts

```diff
@@ -37,6 +37,9 @@ export class UserEntity {
   @Column({ default: false })
   isAdmin!: boolean;
 
+  @Column({ default: true })
+  showOnboarding!: boolean;
```

Comment:
Since this is a per-user field here, are we planning to do onboarding for non-admin users in the future as well?

Reply:
I was also thinking showOnboarding should be a per-instance property. I don't really know how we could currently model this so maybe there is some more refactoring and a re-thinking of entities necessary.

Reply:
There is the metadata database repository, which can store per-instance metadata, like the date of the last reverse geocoding, and whether the onboarding for has been shown or not.